TimoCloud-API/src/main/java/cloud/timo/TimoCloud/api/objects/ProxyGroupObject.java

```diff
@@ -175,7 +189,13 @@
 
     /**
      * Changes the java start parameters
+     *
      * @return A future being completed when the parameters was changed
      */
     APIRequestFuture<Void> setJavaParameters(Collection<String> value);
+
+    /**
+     * @return A String of The JDKPath using to Start the Proxy
+     */
+    String getJdkPath();
```

Comment:
Please replace "jdk" with "jre" here and anywhere else since you don't need the Java Development Kit to run Java applications, but only the Java Runtime Environment (which is included in the JDK though). 
Otherwise, users might get the impression of needing JDK to run TimoCloud.